src/App.js

```diff
@@ -9,7 +10,11 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <Article
+    title={article.title}
```

Comment:
You may use object destructuring here to reduce the amount of code

Reply:
You're right, it will be better))